website/src/css/custom.css

```diff
@@ -256,3 +256,9 @@ table.checkbox-table tbody td {
 .diagonal-cell--bottomLeft {
   grid-column-start: 1;
 }
+
+.migration-guide .typescript-only h4:after {
```

Comment:
:speech_balloon: clever :)

Reply:
taken from the style guide, for consistency 😛